### PR TITLE
Feature #183, added screenshots to install and uninstall instructions

### LINKED ISSUE #183: Add screen shots to installation section of UG
## Describe the Enhancement ##
Add screen shots to the Installation section to help uses new navigate the AWS console. Use [RTDs embedding feature](https://docs.readthedocs.io/en/stable/guides/embedding-content.html#calling-the-embed-api-directly) to pop up the screen shot instead of inserting the image inline with text. 

### Time Estimate ###
1 day

### Sub-Issues ###
Consider breaking the enhancement down into sub-issues.
- [ ] *Add a checkbox for each sub-issue here.*

### Relevant Deadlines ###
*List relevant project deadlines here or state NONE.*

### Assignee ###
- [ ] Select **engineer(s)** or **no engineer** required
- [x] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**

### Projects and Milestone ###
- [x] Select **Project**
- [x] Select **Milestone** as the next official version or **Backlog of Development Ideas**

## Enhancement Checklist ##
- [x] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>/<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)**, **Project**, and **Development** issue
Select: **Milestone** as the next official version
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

### PR DESCRIPTION
**Pull request by georgemccabe**
Closes #183

## Expected Differences ##

- [X] Do these changes modify the system output in any way? **[No]**</br>

## Pull Request Testing ##

- [X] Describe testing already performed for these changes:</br>

Reviewed documentation updates

- [X] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

Review documentation updates and screenshots. Rendered content can be found here:
https://wrfcloud.readthedocs.io/en/feature_183_screenshots/Users_Guide/installation.html

- [X] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes]**

- [X] Do these changes include sufficient testing updates? **[Yes]**

- [X] Will this PR result in changes to the test suite? **[No]**</br>

- [X] Please complete this pull request review by **7/10/2023**.</br>

## Pull Request Checklist ##
- [X] Review the source issue metadata (labels, project, and milestone).
- [X] Complete the PR definition above.
- [X] Ensure the PR title matches the feature or bugfix branch name.
- [X] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**
Select: **Project**
Select: **Milestone** as the version that will include these changes
Select: **Development** to link to the original development issue.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

File: docs/_static/uninstall_5b_delete_email.png
Comment (hahnd) [inline docs/_static/uninstall_5b_delete_email.png:1]: Let's blur-out the email address here.

File: docs/_static/install_12_ami_complete.png
Comment (hahnd) [inline docs/_static/install_12_ami_complete.png:1]: I think AWS typically blurs out account id and usernames in their screen captures.  Probably not detrimental alone, however it does reveal partial login information, which should be kept secret.  Now that this is in the public domain, make sure you have a strong and unique password.

File: docs/_static/install_13a_image_builder.png
Comment (hahnd) [inline docs/_static/install_13a_image_builder.png:1]: blur username/account id

File: docs/_static/install_13b_wrf_pipeline.png
Comment (hahnd) [inline docs/_static/install_13b_wrf_pipeline.png:1]: blur username/account id

File: docs/_static/install_13c_building.png
Comment (hahnd) [inline docs/_static/install_13c_building.png:1]: blur username/account id

File: docs/_static/install_13d_steps.png
Comment (hahnd) [inline docs/_static/install_13d_steps.png:1]: blur username/account id

File: docs/_static/install_9a_select_stack.png
Comment (hahnd) [inline docs/_static/install_9a_select_stack.png:1]: blur username/account id

File: docs/_static/install_9b_select_events.png
Comment (hahnd) [inline docs/_static/install_9b_select_events.png:1]: blur username/account id

File: docs/_static/uninstall_4a_search_ec2.png
Comment (hahnd) [inline docs/_static/uninstall_4a_search_ec2.png:1]: blur username/account id

File: docs/_static/uninstall_4ca_amis.png
Comment (hahnd) [inline docs/_static/uninstall_4ca_amis.png:1]: blur account id

File: docs/_static/uninstall_4db_delete_snapshot.png
Comment (hahnd) [inline docs/_static/uninstall_4db_delete_snapshot.png:1]: blur username/account id

File: docs/_static/uninstall_4ea_delete_ssh.png
Comment (hahnd) [inline docs/_static/uninstall_4ea_delete_ssh.png:1]: blur username/account id

File: docs/_static/uninstall_6a_cf.png
Comment (hahnd) [inline docs/_static/uninstall_6a_cf.png:1]: blur username/account id

**Review by hahnd — CHANGES_REQUESTED** [review]

File: docs/_static/uninstall_4ea_delete_ssh.png
Comment (hahnd) [reply docs/_static/uninstall_4ea_delete_ssh.png:1]: Maybe hide all of our usernames and SSH key fingerprints here too.

File: docs/_static/uninstall_4ca_amis.png
Comment (hahnd) [reply docs/_static/uninstall_4ca_amis.png:1]: blur account id here.  It is in the ARN in the search auto-complete.

File: docs/_static/install_11_login.png
Comment (hahnd) [inline docs/_static/install_11_login.png:1]: blur-out your email address and password length.

**Review by hahnd — CHANGES_REQUESTED** [review]

**Review by hahnd — APPROVED** [review]

**Review by fossell — APPROVED** [review]
The screen shots are a great addition, the text overlay directions are clear and helpful. Looks good!

**Pull request merged**